config/packages.cmake

```diff
@@ -231,6 +231,8 @@ if(ENABLE_PARTITIONING)
 
 endif()
 
+cinch_target_link_libraries(flecsi "${FLECSI_RUNTIME_LIBRARIES};${PARTITION_LIBRARIES};${IO_LIBRARIES}")
```

Comment:
This is because the list of libraries you were linking to was empty.  I added the necessary protection and fixed your unallowable violation of the 80 character limit judiciously enforced by @tuxfan .  I also added a fatal error message to the cinch interface in case a user accidentally does this.